src/test/java/nl/praegus/fitnesse/responders/testHistory/TestHistoryRecentTest.java

```diff
@@ -39,15 +39,17 @@ public void When_directory_is_not_null_return_historylines_not_sorted() {
     }
 
     @Test
-    public void When_dateTimeFormatter_is_not_working_correctly(){
+    public void checks_date_is_correct() throws ParseException {
         RecentTestHistory recentTestHistory = new RecentTestHistory(getMockDir("TestResultDirectory"));
-        String expectedNotEqual = "Wed Jan 22 11:43:40 CET 2020";
-        String expectedContains = "nl.praegus.fitnesse.responders.testHistory.TestHistoryLine@";
 
-        String receivedResult = recentTestHistory.getHistoryLines().toString();
+        SimpleDateFormat isoFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+        isoFormat.setTimeZone(TimeZone.getDefault());
 
-        assertThat(receivedResult).isNotEqualTo(expectedNotEqual);
-        assertThat(receivedResult).contains(expectedContains);
+        Date expectedResult = isoFormat.parse("2020-01-22T11:43:40");
```

Comment:
Ik ben er echt geen fan van dat jullie de Date gebruiken. Je moet echt ZonedDateTime of LocalDateTime gebruiken, dat zijn de java 8+ datetime libraries. Geldt ook voor de code elders.

Reply:
Date is veranderd naar LocalDateTime in de test en in RecentTestHistory.java file.